src/clientManager.js

```diff
@@ -32,12 +32,29 @@ ClientManager.prototype.findClient = function(name) {
 };
 
 ClientManager.prototype.autoloadUsers = function() {
-	this.getUsers().forEach((name) => this.loadUser(name));
+	const users = this.getUsers();
+	const noUsersWarning = `There are currently no users. Create one with ${colors.bold("lounge add <name>")}.`;
+
+	// There was an error, already logged, but we have to crash the server as
+	// user directory could not be accessed
+	if (users === undefined) {
+		process.exit(1);
+	}
+
+	if (!users.length) {
```

Comment:
Can we specify `=== 0` instead of relying on falseyness. We've had a couple of bugs based on falseyness and I think it makes it more clear.